Add a bio file for Ondine

ondine opened the pull request:
This Pull Request adds a bio file for @ondine as requested in #5.

@githubteacher I would :heart: if you would put some :eyes: on the bio and share any :thought_balloon: you have.

Commit:
Add a bio file for Ondine

stevensgill commented:
Your bio looks good to me!

stevensgill commented on docs/ondine.md:
```diff
@@ -0,0 +1,5 @@
+
+- **Name:** Ondine
+- **Occupation:** Technical Writer
+- **Location:** Emeryville
```
Which state is Emeryville in?

Commit:
Specified the correct State for Emeryville

Thanks @stevensgill!  :thumbsup: https://en.wikipedia.org/wiki/Emeryville,_California

ondine commented:
Fixes #5

ondine commented:
Fixes #5
